### PR TITLE
style: Updated skills UI

### LINKED ISSUE #2441: Skills fragment UI
**UI of skills fragment can be improved**
Margin and fonts of few elements are not correct.

**Screenshots**
SkillsOld1
SkillsOld

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by pallav12**
Fixes #2441 
Changes: Added additional cards, Changed font style, Changed margin 

Screenshots for the change: 

SkillsNew

SkillNew1

**GIF**
SkillsUpdatedgif

### CODE REVIEW

**Comment by atm1504** [comment]
Share some screenshots that show the difference between the two states oft he app (Before and after changing code)

**Comment by pallav12** [comment]
Before
WhatsApp Image 2020-01-03 at 23 44 15
After
SkillNew1

before
WhatsApp Image 2020-01-03 at 23 44 14

after
SkillsNew

File: app/src/main/res/layout/item_skill.xml [hunk 1]
```diff
@@ -9,13 +9,14 @@
     <android.support.v7.widget.CardView
         android:layout_width="@dimen/websearch_card_width"
         android:layout_height="wrap_content"
-        android:layout_marginBottom="@dimen/margin_small"
-        android:layout_marginLeft="@dimen/margin_small"
         android:layout_marginStart="@dimen/margin_small"
+        android:layout_marginLeft="@dimen/margin_small"
```
Comment (atm1504) [inline app/src/main/res/layout/item_skill.xml]: Unnecessary change, revert

File: app/src/main/res/layout/item_skill.xml [hunk 2]
```diff
@@ -9,13 +9,14 @@
...
         android:layout_marginTop="@dimen/margin_small"
+        android:layout_marginBottom="@dimen/margin_small"
         android:foreground="?attr/selectableItemBackground"
         app:cardBackgroundColor="@color/susi_message_bg"
         app:cardCornerRadius="@dimen/radius_skill_card_corner"
+        app:cardElevation="4dp"
```
Comment (atm1504) [inline app/src/main/res/layout/item_skill.xml]: Don't hardcode dimens value

File: app/src/main/res/layout/item_skill.xml [hunk 3]
```diff
@@ -68,14 +69,14 @@
                 android:layout_height="wrap_content"
                 android:background="@color/default_bg"
                 android:ellipsize="end"
+                android:fontFamily="sans-serif"
                 android:maxLines="1"
                 android:paddingLeft="@dimen/cmv_padding"
-                android:paddingRight="@dimen/cmv_padding"
                 android:paddingTop="@dimen/padding_small"
+                android:paddingRight="@dimen/cmv_padding"
```
Comment (atm1504) [inline app/src/main/res/layout/item_skill.xml]: Same

File: app/src/main/res/layout/item_skill_group_list.xml [hunk 4]
```diff
@@ -2,23 +2,22 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:id="@+id/group_parent"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
+    android:layout_height="50dp"
```
Comment (atm1504) [inline app/src/main/res/layout/item_skill_group_list.xml]: Dont hard code dimens value

File: app/src/main/res/layout/item_skill_group_list.xml [hunk 5]
```diff
@@ -2,23 +2,22 @@
...
+        android:layout_marginStart="@dimen/margin_small_medium"
         android:layout_weight="1"
         android:gravity="center_vertical"
         android:padding="@dimen/padding_small"
         android:text="@string/sample_group"
-        android:textColor="@color/md_grey_700"
-        android:textSize="@dimen/text_size_moderate"
-        android:textStyle="bold"
-        android:typeface="serif"
-        android:fontFamily="sans-serif"/>
```
Comment (atm1504) [inline app/src/main/res/layout/item_skill_group_list.xml:21]: Why?

**Review by atm1504 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/item_skill_group_list.xml (hunk 5 above)
Comment (pallav12) [reply app/src/main/res/layout/item_skill_group_list.xml:21]: that's because it's better without it... bold font doesn't look good, same with font family, you can see the difference.

File: app/src/main/res/layout/item_skill_group_list.xml (hunk 4 above)
Comment (pallav12) [reply app/src/main/res/layout/item_skill_group_list.xml]: should i create new string... I have to approximate it if i want to use existing

File: app/src/main/res/layout/item_skill.xml (hunk 1 above)
Comment (pallav12) [reply app/src/main/res/layout/item_skill.xml]: sorry, my bad... that was inaccuracy

File: app/src/main/res/layout/item_skill_group_list.xml (hunk 4 above)
Comment (atm1504) [reply app/src/main/res/layout/item_skill_group_list.xml]: Create a new string

**Comment by pallav12** [comment]
Updated according to changes reviewed

File: app/src/main/res/layout/item_skill.xml (hunk 1 above)
Comment (atm1504) [reply app/src/main/res/layout/item_skill.xml]: @pallav12 here I meant to say that the values margin_left and margin_right have not changed, but it is unnecessary. change in position of these attributes. Please correct all these mistakes in the entire file.

Comment (pallav12) [reply app/src/main/res/layout/item_skill.xml]: Okay... Now there's no such change which add +1 in "difference" just because reordering of attributes. Just squashed all commits and force pushed it.

File: app/src/main/res/layout/item_skill_group_list.xml [hunk 6]
```diff
@@ -3,22 +3,22 @@
...
     android:layout_height="wrap_content"
+    android:background="@color/md_white_1000"
     android:orientation="horizontal"
     android:padding="@dimen/padding_small">
 
     <TextView
         android:id="@+id/group"
         android:layout_width="0dp"
-        android:layout_height="wrap_content"
+        android:layout_height="50dp"
```
Comment (atm1504) [inline app/src/main/res/layout/item_skill_group_list.xml]: Don't hardcode dimens value

Comment (pallav12) [reply app/src/main/res/layout/item_skill_group_list.xml]: Sorry... I created dimen but didn't used it, Squashed all commits and updated according to development.

**Review by mohitm15 — COMMENTED** [review]
Is this necessary?

**Comment by pallav12** [comment]
@mohitm15 it's a UI change... It looks more decent(to me) with it, I can close this if it's not adding value.

**Comment by pallav12** [comment]
@atm1504 , @mohitm15 should I close this if it's not adding much value?

**Comment by mohitm15** [comment]
@pallav12 sure 
    Look for any good first issue  if you wanted to start 

_Sent from my VIBE K6 Power using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_
